README.md

```diff
@@ -102,6 +102,28 @@ python add_mmseqs_msa.py --input_json <input_json> --output_json <output_json> -
 - `<custom_template_chain>` : [conditionally required] The chain ID of the chain to use in your custom template, only required if using a multi-chain template. 
 - `<target_id>` : [conditionally required] The ID of the sequence the custom template relates to, only required if modelling a complex.
 
+### Running AlphaFold3
```

Comment:
I really like this inclusion,  does it need to be run with the docker run it? as the alphafold people do below?

```bash
docker run -it \
    --volume $HOME/af_input:/root/af_input \
    --volume $HOME/af_output:/root/af_output \
    --volume <MODEL_PARAMETERS_DIR>:/root/models \
    --volume <DATABASES_DIR>:/root/public_databases \
    --gpus all \
    alphafold3 \
    python run_alphafold.py \
    --json_path=/root/af_input/fold_input.json \
    --model_dir=/root/models \
    --output_dir=/root/af_output```

Reply:
I might add a setup script to add the different python scripts to the local path so that you can run `python setup.py` and then just have it installed.

Reply:
Yes I was thinking about that! Make it into a pip package if you can